documentation/providers/loopia.md

```diff
@@ -0,0 +1,155 @@
+Loopia is an absolutely 💩 provider of DNS API, if you want to use a decent API. 
```

Comment:
```
Loopia is an absolutely 💩 provider of DNS. Using DNSControl hides some of the 💩.
If you are stuck with Loopia, hopefully this will reduce the pain.

They provide DNS services, both as a registrar, and a provider. 
They provide support in English and other regional variants (Norwegian, Serbian, Swedish).

This plugin is based on API documents found at 
[https://www.loopia.com/api/](https://www.loopia.com/api/)
and by observing API responses. Hat tip to github @hazzeh whose code for the
LEGO Loopia implementation was helpful.

Sadly the Loopia API has some problems:
* API calls are limited to 60 calls per minute.  If you go above this,
  you will have to wait before you can make changes.
* When rate-limited, you will not receive a single
  error: The errors propagate from the back-end, with no headers, or
  Retry-After or anything useful.
* Some testing has revealed weird behavior with NS records
* There are no guarantees of idempotency from their API.
```